### PR TITLE
Adds new ear signs

### PR DESCRIPTION
**Pull request by PatrickGoettsch**
## Introduced Changes

This PR shows in the first place the whistle (as usual) and adds the charging status and in case there are no game controller message for 5 seconds, when a first message has arrived. It also shows the maximum temperature on the ears in percent.

Fixes #

## How to Test

Run this with game controller. Let it send message, stop it see if it blinks. See if the whistle works. Let the robot walk and see if the temperature increases.

### CODE REVIEW

File: etc/parameters/default.json
Comment (knoellle) [inline etc/parameters/default.json:1]: Why does this PR change walking parameters?

File: crates/control/src/led_status.rs [hunk 1]
```diff
@@ -171,6 +200,38 @@ impl LedStatus {
...
+                return Ear::full_ears(1.0);
+            } else {
+                return Ear::full_ears(0.0);
+            }
+        }
+
+        let minimum_temperature = 30.0;
+
+        let relative_temperature = (current_maximum_temperature - minimum_temperature)
+            / (105.0 - minimum_temperature).floor();
```
Comment (knoellle) [inline crates/control/src/led_status.rs:230]: You named the minimum temperature constant but not the maximum. Please do so.

**Review by knoellle — CHANGES_REQUESTED** [review]
You seem to have a large squash commit in your branch that includes a lot of changes from other people. Please remove it.

File: etc/parameters/default.json
Comment (PatrickGoettsch) [reply etc/parameters/default.json:1]: Done

File: crates/control/src/led_status.rs (hunk 1 above)
Comment (PatrickGoettsch) [reply crates/control/src/led_status.rs]: Done

**Review by knoellle — APPROVED** [review]
LGTM

**Pull request merged**